ogc-example/src/main/resources/logback.xml

```diff
@@ -5,7 +5,9 @@
     </encoder>
   </appender>
 
-  <logger name="geotrellis.server.ogc" level="${GT_OGC_LOG_LEVEL:-INFO}"/>
+  <logger name="geotrellis.store" level="${GT_STORE_LOG_LEVEL:-WARN}"/>
+  <logger name="geotrellis.server.ogc" level="${GT_OGC_LOG_LEVEL:-DEBUG}"/>
+  <logger name="org.http4s" level="${HTTP4S_LOG_LEVEL:-WARN}"/>
```

Comment:
Can we include new params descritpion into https://github.com/geotrellis/geotrellis-server/blob/develop/ogc-example/docs/usage.md?